scripts/ci-integration.sh

```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
-# shellcheck source=../hack/ensure-go.sh
+# shellcheck source=/dev/null
```

Comment:
I would expect this to point to an actual file not /dev/null

Reply:
Same comment applies to other places where `shellcheck source=/dev/null`

Reply:
I was doing this wrong. I've fixed it now.